src/controllers/dynakube/deploymentmetadata/config.go

```diff
@@ -0,0 +1,28 @@
+package deploymentmetadata
+
+import (
+	"github.com/Dynatrace/dynatrace-operator/src/logger"
+)
+
+const (
+	orchestrationTech = "Operator"
+
+	keyOperatorScriptVersion = "script_version"
+	keyOrchestratorID        = "orchestrator_id"
+	keyOrchestrationTech     = "orchestration_tech"
+
+	DeploymentTypeApplicationMonitoring = "application_monitoring"
+	DeploymentTypeFullStack             = "classic_fullstack"
```

Comment:
`ClassicFullStack`, or just `Classic`

Reply:
we have to be careful here, changing this would lead to different values in cluster-data-hub

Reply:
I meant the constant name, not the value 😄